doc/article/en-US/dialog-basics.md

```diff
@@ -75,18 +75,21 @@ typings install github:aurelia/dialog --save
 
 <code-listing heading="main.js">
   <source-code lang="ES 2015">
+  import {PLATFORM} from 'aurelia-pal';
   export function configure(aurelia) {
     aurelia.use
       .standardConfiguration()
       .developmentLogging()
-      .plugin('aurelia-dialog');
+      .plugin(PLATFORM.moduleName('aurelia-dialog'));
 
     aurelia.start().then(a => a.setRoot());
   }
   </source-code>
 </code-listing>
 
-> Note: If you are using WebPack it is possible that the plugin is installed before Aurelia has replaced the `<body>` element, if that is where your `aurelia-app="main"` is defined, which results in some of the dialog components getting overwritten.  In this case you can move the `aurelia-app` attribute to a `<div>` inside of the `<body>`.  Example - `<body><div aurelia-app="main"></div></body>`.
+> Note: If you are using Webpack it is possible that the plugin is installed before Aurelia has replaced the `<body>` element, if that is where your `aurelia-app="main"` is defined, which results in some of the dialog components getting overwritten.  In this case you can move the `aurelia-app` attribute to a `<div>` inside of the `<body>`.  Example - `<body><div aurelia-app="main"></div></body>`.
```

Comment:
Reading this again and it looks nasty to me.
Isn't that a timing bug in `aurelia-dialog`?
If you put stuff inside the `<body>` couldn't you just wait on Aurelia's loaded event?